src/single-spa-vue.js

```diff
@@ -141,8 +151,12 @@ function update(opts, mountedInstances, props) {
 function unmount(opts, mountedInstances, props) {
   return Promise.resolve().then(() => {
     const instance = mountedInstances[props.name];
-    instance.vueInstance.$destroy();
-    instance.vueInstance.$el.innerHTML = "";
+    if (opts.createApp) {
+      instance.vueInstance.unmount(instance.domEl);
```

Comment:
Vue 3 is better at unmounting! No need to clear out the innerHTML ourselves, and the router reportedly unmounts correctly in Vue 3